cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -13,6 +13,8 @@
 -- not export any from this API. We also use them unticked as nature intended.
 {-# LANGUAGE DataKinds #-}
 {-# OPTIONS_GHC -Wno-unticked-promoted-constructors #-}
+{-# LANGUAGE BangPatterns #-}
```

Comment:
These pragmas should be above and is BlockArguments actually in use here?

Reply:
Nope.  Just for temporary type checking purposes.  Removing.